screens/CheckoutScreen.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { BASE } from "../lib/common";
+
+import {
+  AsyncStorage,
+  StyleSheet,
+  TextInput,
+  View,
+  Button,
+  Text
+} from "react-native";
+
+export default class CheckoutScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      customerName: "",
+      customerId: 0,
+      currPoints: 0,
+      cart: [],
+      totalPrice: 0,
+      totalPoints: 0
+    };
+  }
+
+  async componentDidMount() {
+    const customerId = await AsyncStorage.getItem("customerId");
+    this.setState({ customerId: customerId });
+
+    this.getUser("Customers", customerId).then(customerRecord => {
+      if (customerRecord) {
+        let currPoints = customerRecord["fields"]["points"];
+        let name =
+          customerRecord["fields"]["First Name"] +
+          " " +
+          customerRecord["fields"]["Last Name"];
+        this.setState({ customerName: name, currPoints: currPoints });
+      }
+    });
+  }
+
+  async getUser(table, id) {
+    return BASE(table).find(id);
+  }
+
+  updateCustomerPoints() {
```

Comment:
For example, this could take in parameters `this.state.customerId`, `this.state.currPoints`, `this.state.totalPoints` as (`id`, `oldPoints`, `totalPoints`)

Instead of doing `function(err)`, you could do `.then().catch(err => console.error(err))`
I don't recall what update returns, so I forget what you would want to do with the retval (if any exists) but yeah!

Reply:
Removing this file